### PR TITLE
Fix typo in S.Diagnostics.Tracing csproj

### PR DESCRIPTION
**Pull request by danmosemsft**
I have no idea what this define does but I assume this wasn't intended.

### CODE REVIEW

File: src/System.Diagnostics.Tracing/src/System.Diagnostics.Tracing.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <!-- CSC adds a typeforward for the internal nested type EventSource.EventMetadata but
          GenFacades doesn't consider this as a viable target for a typeforward since it's internal -->
     <GenFacadesIgnoreMissingTypes Condition="'$(TargetGroup)' == 'netfx'">true</GenFacadesIgnoreMissingTypes>
-    <DefineContants Condition="'$(TargetGroup)' != 'netfx'">$(DefineConstants);SUPPORTS_EVENTCOMMANDEXECUTED</DefineContants>
+    <DefineConstants Condition="'$(TargetGroup)' != 'netfx'">$(DefineConstants);SUPPORTS_EVENTCOMMANDEXECUTED</DefineConstants>
```
Comment (am11) [inline src/System.Diagnostics.Tracing/src/System.Diagnostics.Tracing.csproj]: Same typo also exists under https://github.com/dotnet/corefx/blob/c258841/Documentation/coding-guidelines/project-guidelines.md#define-naming-convention.

Comment (jkotas) [inline src/System.Diagnostics.Tracing/src/System.Diagnostics.Tracing.csproj]: This whole line be deleted. This .csproj is a pure façade for everything bug netfx.

**Comment by danmosemsft** [comment]
I'm going to abandon this as @brianrob is going to make larger changes here.

File: src/System.Diagnostics.Tracing/src/System.Diagnostics.Tracing.csproj (hunk 1 above)
Comment (jkotas) [inline src/System.Diagnostics.Tracing/src/System.Diagnostics.Tracing.csproj]: #18524

**Pull request closed without merging**